.travis.yml

```diff
@@ -8,9 +8,13 @@ jobs:
         - docker
       before_install:
         - sudo apt install jq
+        - chmod +x ./e2e-tests/stream-mode-tests.sh
       script:
         - mvn clean install
         - ./e2e-tests/batch-mode-tests.sh
+      script:
+        - mvn clean install
```

Comment:
@gitcliff  this looks like a duplicate ,
why run mvn clean install the second time